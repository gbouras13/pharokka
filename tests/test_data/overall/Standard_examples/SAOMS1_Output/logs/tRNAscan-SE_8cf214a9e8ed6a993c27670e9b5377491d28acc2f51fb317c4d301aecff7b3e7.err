
tRNAscan-SE v.2.0.12 (Nov 2022) - scan sequences for transfer RNAs
Copyright (C) 2022 Patricia Chan and Todd Lowe
                   University of California Santa Cruz
Freely distributed under the GNU General Public License (GPLv3)

------------------------------------------------------------
Sequence file(s) to search:        tests/test_data/overall/Standard_examples/SAOMS1.fasta
Search Mode:                       General
Results written to:                Standard output
Output format:                     Tabular
Searching with:                    Infernal single-pass scan
                                   Fast mode
Covariance model:                  /Users/a1667917/mambaforge/envs/pharokka_env/lib/tRNAscan-SE/models/TRNAinf.cm

Temporary directory:               /var/folders/t9/ns8v1yq93_j7nhlnbcntkp6c0000gq/T/
tRNA predictions saved to:         SAOMS1_Output/trnascan_out.gff
------------------------------------------------------------

Command line: tRNAscan-SE --thread 8 -G -Q -j SAOMS1_Output/trnascan_out.gff tests/test_data/overall/Standard_examples/SAOMS1.fasta
