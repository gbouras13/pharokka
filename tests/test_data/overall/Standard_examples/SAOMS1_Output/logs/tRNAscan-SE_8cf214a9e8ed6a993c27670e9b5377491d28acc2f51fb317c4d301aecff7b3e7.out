Status: Running Infernal analysis
Sequence   		tRNA  	Bounds	tRNA	Anti	Intron Bounds	Inf	      
Name       	tRNA #	Begin 	End   	Type	Codon	Begin	End	Score	Note
--------   	------	----- 	------	----	-----	-----	----	------	------
MW460250_1 	1	138642	138569	Asp	GTC	0	0	58.8	
MW460250_1 	2	138562	138490	Phe	GAA	0	0	67.8	
MW460250_1 	3	115265	115194	Met	CAT	0	0	61.2	
